src/AddTodoForm.jsx

```diff
@@ -8,11 +8,53 @@ export default function AddTodoForm({ onAddTodo }) {
 		let newTodoTitle = event.target.value;
 		setTodoTitle(newTodoTitle);
 	}
+
+	const postTodo = async (newTodo) => {
```

Comment:
This is great! I'm glad you pushed yourself to complete the stretch goal!! My only feedback on this is that it would be better to put it in the `addTodo` function that's passed into this component. Posting the data to the backend is something that the parent component should be responsible for, not this component (which is more of a "display" component than a "smart" component). This is still great work. Good job, Olga!

Reply:
ok thanx!! I'll do it!)